alibi_detect/cd/pytorch/lsdd.py

```diff
@@ -165,3 +174,24 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray]:
 
         p_val = (lsdd <= lsdd_permuted).float().mean()
         return float(p_val.cpu()), float(lsdd.cpu().numpy()), lsdd_permuted.cpu().numpy()
+
+    def get_config(self) -> dict:
+        """
+        Get the detector's configuration dictionary.
+
+        Returns
+        -------
+        The detector's configuration dictionary.
+        """
+        cfg = super().get_config()
+
+        # backend
+        cfg.update({'backend': 'pytorch'})
```

Comment:
I thought the `backend` kwarg was set as an attribute of the backend class by the agnostic class, hence wouldn't need to be hard-coded here? (same comment applies to `mmd` etc. below).

Reply:
This wasn't actually the original intention (the `self.backend` was intended for use at save time, in the later PR). 

However, this is actually a nice idea, since we can then move the `cfg.update({'backend': self.backend})` up to the base class.